### ISSUE TITLE
Angular AIO General Setup for Authoring - "yarn setup" fails

### ISSUE THREAD

**Issue by FrederikSchlemmer** [angular/angular#25759]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[x] Other... Please describe:
The executed "yarn setup" command cannot be completed and terminates with an error during the creation of the documentation.
</code></pre>

## Current behavior

The current behavior is that after the conversion of the necessary symlinks (chapter: Developing under Windows) and the execution of 'yarn' the following command 'yarn setup' fails.
Furthermore, the 'yarn docs' command also fails with the same error.
From this I conclude that the problem arises during the generation of the documentation, perhaps?

![image](https://user-images.githubusercontent.com/35023083/44902326-f914c900-ad09-11e8-82a6-77ed1c78c03e.png)

## Expected behavior

The steps given for the 'General Setup' under 'Guide to authoring' should be straightforward.

## Minimal reproduction of the problem with instructions

To reproduce the error, the following steps must be carried out:
- Copying the current master branch
- Creating the suitable Windows symlinks
- Work through the 'General Setup' under 'Guide to authoring'

## What is the motivation / use case for changing the behavior?

Ability to test contributions before a pull request is submitted.

## Environment

<pre><code>
Angular version: Latest Repo version

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: v8.11.2  
- Yarn version: v1.7.0
- Platform: Windows 10  

Others:

</code></pre>

**Comment by brandonroberts** [comment]
Run `yarn` in the root of the project first before going into the `aio` folder and running `yarn`, `yarn setup`, etc..

**Comment by FrederikSchlemmer** [comment]
@brandonroberts I think this part should be better documented. The step with the "yarn" in the root folder is not mentioned anywhere.

Or should the sentence "ensure the dependencies are installed" reference it?

**Comment by brandonroberts** [comment]
I agree as it wasn't previously required to install dependencies at the root also. I think @petebacondarwin is fixing this so you don't have to explicitly do that.

**Comment by petebacondarwin** [comment]
I'll add a `yarn install` at the root level into the `yarn setup` in the aio folder.
